plaidbench/plaidbench/core.py

```diff
@@ -309,8 +312,7 @@ def _inner_run(reports,
         if gflops:
             resstr += ', {:.2f} (GFLOP/s)'.format(gflops)
         click.secho(resstr, fg='cyan', bold=True)
-        if frontend.name == 'plaidml' and 'metal' in str(device):
-            tile_exec_per_example = exec_per_example
```

Comment:
This was a workaround for `metal`, now I can't remember what it was for. I suppose we can revist this when we get Metal back.

Reply:
Let me know if it should be put back.  Seemed redundant.